src/App.vue

```diff
@@ -87,6 +88,9 @@ export default {
     setDataValue(event) {
       event[0].value = event[1];      
     },
+    setDataList(event) {
+      event[0].list = event[1];      
```

Comment:
no ked to srovnavam s tym nad tym tak mozno by to slo mergnut do jednej metody s tym ze tam predavas `event` a `key` ktory modifikujes
takze telo by potom vypadalo `event[0][key] = event[1]`

Reply:
ale asi to v tomto stave teraz nema zmysel, mozno keby sa tam pridal nejaky treti kluc... alebo to mozno nerobit ako metodu ale normalne tam flaknut to nastavenie hodnoty...